packer/jenkins-agent-win2016-x64.json

```diff
@@ -0,0 +1,86 @@
+{
+  "variables":{
+    "name-base":"Jenkins-Agent-Windows2016",
+    "build-region":"us-east-1",
+    "build-vpc":"vpc-<>",
+    "build-subnet":"subnet-<>",
+    "build-secgrp":"sg-<>",
+    "build-time":"{{isotime \"2006-01-02T03-04-05Z\"}}",
+    "aws_ami_region":"us-east-1"
+  },
+  "builders":[
+    {
+      "name":"{{user `name-base`}}-{{user `build-time`}}" ,
+      "ami_description":"{{user `name-base`}}",
+      "type":"amazon-ebs",
+      "encrypt_boot":"false",
+      "region":"{{user `build-region`}}",
+      "ami_regions":"{{user `aws_ami_region`}}",
+      "instance_type":"c5.4xlarge",
+      "ami_name":"{{user `name-base`}}-{{user `build-time`}}",
+      "vpc_id":"{{user `build-vpc`}}",
+      "subnet_id":"{{user `build-subnet`}}",
+      "security_group_ids":"{{user `build-secgrp`}}",
+      "source_ami_filter":{
+        "filters":{
+          "virtualization-type":"hvm",
+          "architecture":"x86_64",
+          "name": "Windows_Server-2016-English-Full-Base-20*",
+          "block-device-mapping.volume-type":"gp2",
```

Comment:
Using `gp3` is better.

Reply:
gp3 has lag / latency in practice tho I would put this to gp2 for now.
Pretty easy to rebuild right now with packer.
gp2 can ebs optimize to 900 IOPS which is not bad during jenkins test.